### PR TITLE
Add functional set cover trainer

### PR DESCRIPTION
**Pull request by alexcdot**
- Added set cover that finds 10% of training set that best covers the rest in terms of their activations with the model
- Added fixed train and validation set files and loading
- Also added random subset trainer

### CODE REVIEW

**Comment by alexcdot** [comment]
Merging because of efficiency, but PR is here so you can easily see the things i changed
